packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,248 @@
+  // because it's possible for the access token to be invalidated when we display the avatar image.
+  // if we get the notification of a change, we re-render the card.
+
+  const onSubmit = useCallback(async () => {
+    try {
+      isPasswordNeeded
+        ? signInWithPassword(email, password)
+        : signInUsingLoggedInAccount();
+      // TODO: add message in Alert Bar re: success, then navigate to where appropriate
+    } catch (e) {
+      // TODO: metrics event for error
+      // TODO: Add in localized Alert Bar message for error
+      setError(e);
+    }
+  }, [
+    email,
+    password,
+    signInUsingLoggedInAccount,
+    signInWithPassword,
+    isPasswordNeeded,
+    setError,
+  ]);
+
+  // TODO:
+  // Add in the Alert Bar in place of the original `success` and `error` display divs
+
+  return (
+    <>
+      <div className="mb-4">
+        {isPasswordNeeded ? (
+          <FtlMsg id="signin-password-needed-header">
+            <h1 className="card-header">
+              Enter your password{' '}
+              <span className="card-subheader">for your Firefox account</span>
+            </h1>
+          </FtlMsg>
+        ) : (
+          <>
+            <FtlMsg id="signin-header">
+              <h1 className="card-header">Sign in</h1>
+            </FtlMsg>
+            <>
+              {isPocketClient && (
+                <FtlMsg id="signin-subheader-with-logo">
+                  <span className="card-subheader">
+                    Continue to{' '}
+                    <span>
+                      <PocketLogo
+                        className="inline"
+                        aria-label={localizedPocketAriaLabel}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && serviceLogo && ServiceLogo && (
+                <FtlMsg id="signin-subheader-with-logo">
+                  <span className="card-subheader">
+                    Continue to{' '}
+                    <span>
+                      <ServiceLogo
+                        className="inline"
+                        aria-label={localizedLogoAriaLabel}
+                      />
+                    </span>
+                  </span>
+                </FtlMsg>
+              )}
+              {!isPocketClient && !serviceLogo && (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          <div className="avatar-wrapper mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
```

Comment:
Looks like you can remove the `avatar-wrapper` class.

I think we can actually just use the `Avatar` component from `fxa-settings` when we get this to the functional stage. Mind adding a comment about that?